fr/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -3,47 +3,205 @@ id: applications-databases-elasticsearch
+| docs_count               | Nombre de documents                        |         |
+| size_in_bytes            | Tailles de toutes les partitions assignées |    B    |
+
+<!--indice-statistics-->
+
+| Métrique           | Description                                            |  Unité   |
+| :----------------------- | :----------------------------------------------- | :------ |
+| display                  | Nom de l'Indice Elasticsearch                     |         |
+| status                   | Status de l'Indice Elasticsearch                     |         |
+| Shards_active            | Nombre de partitions actives                     |         |
+| Shards_unassigned        | Nombre de partitions non-assignées               |         |
+| docs_count               | Nombre de documents                              |         |
+| size_in_bytes_primaries  | Taille de toute les partitions primaires         |    B    |
+| size_in_bytes_total      | Taille totale de toutes les partitions assignées |    B    |
+
+<!--license-->
+
+| Métrique           | Description                              | Unité   |
+| :----------------- | :--------------------------------------- | :------ |
+| type               | Type de Licence                          |         |
+| status             | Status de la Licence                     |         |
+| issued_to          | Propriétaire de la licence               |         |
+| issue_date         | Date de concession de la licence         |         |
+
+
+<!--node-statistics-->
+
+| Métrique          | Description                                               | Unité   |
+| :---------------- | :-------------------------------------------------------- | :-----  |
+| display           | Nom du noeud Elasticsearch                                |         |
+| indices_count     | Nombre d'indices dans le noeud                            |         |
+| heap_used_percent | Pourcentage de mémoire actuellement utilisée dans la pile |    %    |
+| heap_used_in_bytes| Mémoire disponible pour la pile                           |    B    |
+| heap_max_in_bytes | Taille maximum de mémoire disponible pour la pile         |    B    |
+| available_in_bytes| Nombre total d'octets disponibles                         |    B    |
+| total_in_bytes    | Taille totale de tous les fichiers stockés                |    B    |
+| docs_count        | Nombre de documents dans l'indice                         |         |
+| size_in_bytes     | Taille totale de toutes les partitions assignées au noeud |    B    |
+
+<!--Rules-->
+
+| Non de la rêgle                        | Description                                       |
+| :------------------------------------- | :------------------------------------------------ |
+| App-DB-Elasticsearch-Indice-Statistics | Découverte des indices sur votre BD Elasticsearch |
+| App-DB-Elasticsearch-Node-Statistics   | Découverte des noeuds sur votre BD Elasticsearch  |  
```

Comment:
Same remark for French version.
Put this section under : ### Plugin-Pack assets